plain-pie/src/main/java/az/plainpie/animation/PieAngleAnimation.java

```diff
@@ -0,0 +1,31 @@
+package az.plainpie.animation;
+
+import android.view.animation.Animation;
+import android.view.animation.Transformation;
+
+import az.plainpie.PieView;
+
+/**
+ * @author Alejandro Zürcher (alejandro.zurcher@gmail.com)
+ */
+public class PieAngleAnimation extends Animation {
+
+    private PieView arcView;
+
+    private float oldAngle;
+    private float newAngle;
+
+    public PieAngleAnimation(PieView arcView, int newAngle) {
+        this.oldAngle = arcView.getPieAngle();
+        this.newAngle = newAngle;
+        this.arcView = arcView;
+    }
+
+    @Override
+    protected void applyTransformation(float interpolatedTime, Transformation transformation) {
+        float angle = 0 + ((newAngle - oldAngle) * interpolatedTime);
```

Comment:
I think this could also work if we just do:
`float angle = oldAngle * interpolatedTime;`

Which would also imply removing the `int newAngle` parameter from constructor and variable of class.

I could add in a following commit a change to select the orientation of the fill, by updating the polarity sign of the angle float. I'd add a parameter to the constructor that'd be `boolean shouldFillCounterClockwise` that if true it will set the polarity of `float angle` to '-' otherwise, by default it would be clockwise and the polarity sign would be '+'.

Sounds good?